README.md

```diff
@@ -1,10 +1,47 @@
 # FairCMakeModules
+
 CMake Modules developed in the context of various FAIR projects
 
+## Installation
+
+```
+git clone https://github.com/FairRootGroup/FairCMakeModules
+cmake -S FairCMakeModules -B FairCMakeModules_build -DCMAKE_INSTALL_PREFIX=FairCMakeModules_install
+cmake --build FairCMakeModules_build --target install
+```
+
+## Usage
+
+In your project, discover this package with [`find_package`](https://cmake.org/cmake/help/latest/command/find_package.html#full-signature-and-config-mode):
+
+```
+find_package(FairCMakeModules)
```

Comment:
Maybe with version and/or `REQUIRED`?

Reply:
`REQUIRED` I will add, version and other options user can find in the linked cmake docs.

Reply:
added `REQUIRED` and a note about more options